plexapi/server.py

```diff
@@ -322,7 +322,7 @@ def installUpdate(self):
             # figure out what method this is..
             return self.query(part, method=self._session.put)
 
-    def history(self, maxresults=9999999, mindate=None, ratingKey=None):
+    def history(self, maxresults=9999999, mindate=None, ratingKey=None, accountID=None, librarySectionID=None):
```

Comment:
`maxresults=9999999` causes the search to take a long time. Extending `maxresults=9999999` out to the other history functions would allow for the user to specify the `maxresults` ie: `user.history(maxresults=5)`. Unless there is a better way?